packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -396,25 +399,27 @@ const StaticDesignPicker: React.FC< StaticDesignPickerProps > = ( {
 				/>
 			) }
 			<div className={ 'design-picker__grid' }>
-				{ filteredDesigns.map( ( design ) => (
-					<DesignButtonContainer
-						key={ design.slug }
-						design={ design }
-						locale={ locale }
-						onSelect={ onSelect }
-						onPreview={ onPreview }
-						onUpgrade={ onUpgrade }
-						premiumBadge={ premiumBadge }
-						highRes={ false }
-						hideFullScreenPreview={ false }
-						hideDesignTitle={ false }
-						isPremiumThemeAvailable={ isPremiumThemeAvailable }
-						previewOnly={ previewOnly }
-						hasDesignOptionHeader={ hasDesignOptionHeader }
-						onCheckout={ onCheckout }
-						verticalId={ verticalId }
-						hasPurchasedTheme={ wasThemePurchased( purchasedThemes, design ) }
-					/>
+				{ filteredDesigns.map( ( design, index ) => (
+					<Fragment key={ design.slug }>
+						{ index === 3 ? buildYourOwnCta : null }
```

Comment:
[nit] Same here, we can just short-circuit this.

Reply:
The CTA won't show if there are less than 3 filtered results 😢
![Screen Shot 2022-08-24 at 4 50 58 PM](https://user-images.githubusercontent.com/797888/186374639-82e4a5c9-4fc3-4a24-88d3-36cd607276e6.png)

Reply:
I applied a fixed, but it seems the `100vh` trick to prevent page jump has compromised the position :(

https://d.pr/i/kpep7t

Reply:
I just created an issue for that https://github.com/Automattic/wp-calypso/issues/66905

Reply:
So we can ignore the glitch on this PR